lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -137,8 +147,9 @@ public static function find(array $filters = []) {
    * Build out the instantiated Reportback class object with supplied data.
    *
    * @param object $data
+   * @param bool  $full Boolean to decide whether to fetch full user data.
    */
-  private function build($data) {
+  private function build($data, $full = true) {
```

Comment:
We should probs default this to `false` now that we have a URL parameter to control it, and since the resource file also defaults it to false in the services specification array.